src/Service/AnrService.php

```diff
@@ -992,7 +992,7 @@ public function duplicateAnr(
                 $newInstanceRisk->set('id', null);
                 $newInstanceRisk->setAnr($newAnr);
                 if ($instanceRisk->amv) {
-                    $newInstanceRisk->setAmv($amvsNewIds[$instanceRisk->amv->id]);
+                    $newInstanceRisk->setAmv($amvsNewIds[$instanceRisk->amv->getUuid()->toString()]);
```

Comment:
It's safe if we do (string) typecast instead to be sure the method getUuid() does not return string value.
P.S. there is a task to move the typecast inside of the getUuid() method, then we will not need any typecasting.